impl/src/main/java/org/jboss/weld/bootstrap/events/ProcessSimpleInjectionTarget.java

```diff
@@ -45,6 +45,7 @@ public ProcessSimpleInjectionTarget(BeanManagerImpl beanManager, AnnotatedType<X
     public void setInjectionTarget(InjectionTarget<X> injectionTarget) {
         checkWithinObserverNotification();
         this.injectionTarget = injectionTarget;
+        BootstrapLogger.LOG.setInjectionTargetCalled(getReceiver().getClass(), injectionTarget.getClass());
```

Comment:
Ok if we pass only `getReceiver()` for extension param then we wiill display something like `com.sun.faces.application.view.ViewScopeExtension@69309707` that's not so nice. Same for annotatedType etc.